exaudfclient/base/javacontainer/script_options/test/swig_factory_test.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SWIG_FACTORY_TEST_H
+#define SWIG_FACTORY_TEST_H 1
+
+#include <string>
+#include <map>
+#include "base/swig_factory/swig_factory.h"
+#include <functional>
+
+struct SwigFactoryTestImpl : public SWIGVMContainers::SwigFactory {
```

Comment:
And, a thought I had the factory we give the parser should generate a object that is only able return module content this way we can reuse it for the potential new client